libs/indibase/inditelescope.cpp

```diff
@@ -682,7 +682,9 @@ void Telescope::NewRaDec(double ra, double dec)
         TrackStateSP.apply();
     }
 
-    if (std::abs(EqNP[AXIS_RA].getValue() - ra) > EQ_NOTIFY_THRESHOLD ||
+    // RA is in hours, so change the arc-second threshold accordingly.
+    constexpr double RA_NOTIFY_THRESHOLD = EQ_NOTIFY_THRESHOLD/15.0;
+    if (std::abs(EqNP[AXIS_RA].getValue() - ra) > RA_NOTIFY_THRESHOLD ||
```

Comment:
Why introduce a variable? can't you just divide EQ_NOTIFY_THRESHOLD by 15.0 in the if statement?

Reply:
A constexpr isn't a variable, it's a constant done at compile time. If you prefer I can switch this to the divide by 15. I thought it was more readable this way. Your preference.